integrations/snowflake/src/haystack_integrations/components/retrievers/snowflake/snowflake_table_retriever.py

```diff
@@ -82,11 +84,16 @@ def __init__(
         :param db_schema: Name of the schema to use.
         :param warehouse: Name of the warehouse to use.
         :param login_timeout: Timeout in seconds for login. By default, 60 seconds.
+        :param private_key_file: Location of private key
```

Comment:
this docstring is very confusing - what's `key_file`? can you maybe rephrase it?

Reply:
I chose to match the snowflake connector params to keep it consistent with the snowflake documentation, see sample code here:
- https://docs.snowflake.com/developer-guide/python-connector/python-connector-connect#using-key-pair-authentication-and-key-pair-rotation

I've added some further comments to the docstring to include a note to the following link:
- https://docs.snowflake.com/en/user-guide/key-pair-auth

Reply:
I can't see your changes - also my comment was not about the params names themselves, rather the docstring explaining what they are used for.